mlrun/runtimes/base.py

```diff
@@ -1519,6 +1523,17 @@ class BaseRuntimeHandler(ABC):
     class_modes: typing.Dict[RuntimeClassMode, str] = {}
     wait_for_deletion_interval = 10
 
+    @staticmethod
+    def get_k8s():
```

Comment:
why not private? is it used externally?

Reply:
also since BaseRuntime has the same method shouldn't this be in utils or something?